src/NHibernate.Test/NHSpecificTest/NH3931/Entities.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NHibernate.Test.NHSpecificTest.NH3931
```

Comment:
Could you please move tests to `NHibernate.Test.Insertordering` where the old one lives?